tmt/steps/provision/__init__.py

```diff
@@ -285,14 +285,62 @@ class CheckRsyncOutcome(enum.Enum):
 T = TypeVar('T')
 
 
-class GuestCapability(enum.Enum):
-    """ Various Linux capabilities """
+class GuestCapability(enum.IntEnum):
+    """
+    Linux capabilities as defined in capabilities.h.
+
+    Each enum member represents a Linux capability that can be granted to processes.
+    The integer value of each member corresponds to its bit position in the kernel's
+    capability bitmask, as defined in the Linux kernel's capabilities.h.
+
+    These values are stable across kernel versions as they are part of the kernel's
+    ABI guarantee. New capabilities can only be added with new numbers at the end.
+
+    See: include/uapi/linux/capability.h in the Linux kernel source.
```

Comment:
We have an actual, official link: https://github.com/torvalds/linux/blob/master/include/uapi/linux/capability.h

Reply:
Addressed in 62ad828b667fc6680072f375e6bb30edff9725a9